filter.md

```diff
@@ -0,0 +1,217 @@
+# Filtering the output
+
+Tiger generates a lot of reports. This guide explains how to use filters to control what reports are printed.
+
+In the `ck3-tiger.conf` file, add a `filter` block. Inside the `filter`, add a `trigger` block.
+
+```
+# You can have at most 1 filter block.
+filter = {
+    # Whether to report about problems in vanilla game files.
+    # Setting this to 'yes' results in a LOT of spam.
+    # Optional boolean value, defaults to no.
+    show_vanilla = no
+
+    # Whether to report about problems in mods loaded via the load_mod sections.
+    # Optional boolean value, defaults to no.
+    show_loaded_mods = no
+    
+    # You can have at most 1 trigger block.
+    trigger = {
+        # The root trigger block is an AND-gate, just like in paradox script.
+    }
+}
+```
+
+## Triggers
+
+This is the list of valid triggers that can be used in the `trigger` block.
+
+### always
+
+- `always = yes` matches all reports
+- `always = no` matches no reports
+
+### Logic gates
+
+You can use any of the following logic gates. The `NOT` gate must contain exactly one trigger. All other gates take zero
+or more triggers.
+
+- `NOT = { <trigger> }`
+- `AND = { <trigger> <trigger> ... }`
+    - Matches reports that match all children.
+    - An `AND` block without children is the same as `always = yes`.
+- `OR = { <trigger> <trigger> ... }`
+    - Matches reports that match at least one child.
+    - An `OR` block without children is the same as `always = no`.
+- `NAND = { <trigger> <trigger> ... }`
+    - This is the same as writing `NOT = { AND = { } }`
+- `NOR = { <trigger> <trigger> ... }`
+    - This is the same as writing `NOT = { OR = { } }`
+
+### Severity
+
+Severity is a measure of how serious the problem being reported about is. For example; using three spaces of indentation
+instead of four is an extremely minor problem that wouldn't impact the functionality of your mod at all. It would be a
+very low severity. Conversely, a problem that causes the game to crash on start-up should receive the highest severity.
+
+These are the current severity levels:
+
+- `Tips` These are things that aren't necessarily wrong, but there may be a better, more idiomatic way to do it. This may also include performance issues.
+- `Untidy` This code smells. The player is unlikely to be impacted directly, but developers working on this codebase will likely experience maintenance headaches.
+- `Warning` This will result in glitches that will noticeably impact the player's gaming experience. Missing translations are an example.
+- `Error` This code probably doesn't work as intended. The player may experience bugs.
+- `Fatal` This is likely to cause crashes.
+
+You can add the severity trigger like so:
+
+- `severity == Tips` (Only match tips.)
```

Comment:
Here's where you're using == in examples

Reply:
Is that a problem? Both are supported. Might as well use the more correct one in the examples. Unless you really hate it, I can change it.